atom/browser/api/atom_api_app.cc

```diff
@@ -215,6 +216,15 @@ void App::SetAppUserModelId(const std::string& app_id) {
 #endif
 }
 
+void App::RegisterStandardSchemes(const std::vector<std::string>& schemes) {
+  std::stringstream stream;
+  auto vectorToString = [&stream](const std::string& scheme) {
+    stream << scheme << ",";
+  };
+  std::for_each(schemes.begin(), schemes.end(), vectorToString);
```

Comment:
There is `base::JoinString`.

The string concatenating should be put in `AtomBrowserClient::SetCustomSchemes`, and `AtomBrowserClient::SetCustomSchemes` should take `vector` as parameter.